Add hint for dca mode constants and update note concerning child_record callback (#941)
contao/docs: docs/dev/reference/dca/list.md

| Key                   | Value                            | Description                                                                                                                                                                                                                                                                                                                                                                                            |
|-----------------------|----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | mode                  | Sorting mode (`integer`)         | **0** Records are not sorted <br>**1** Records are sorted by a fixed field <br>**2** Records are sorted by a switchable field <br>**3** Records are sorted by the parent table <br>**4** Displays the child records of a parent record (see style sheets module) <br>**5** Records are displayed as tree (see site structure) <br>**6** Displays the child records within a tree structure (see articles module)<br><br>__Note:__ Before **Contao 4.13**  you will need to implement a [child_record_callback](../callbacks#list-sorting-child-record) in **mode 4**, otherwise no records will be shown in backend.               |

Reviewer: ```suggestion
| mode                  | Sorting mode (`integer`)         | **0** Records are not sorted <br>**1** Records are sorted by a fixed field <br>**2** Records are sorted by a switchable field <br>**3** Records are sorted by the parent table <br>**4** Displays the child records of a parent record (see style sheets module) <br>**5** Records are displayed as tree (see site structure) <br>**6** Displays the child records within a tree structure (see articles module)<br><br>__Note:__ Before Contao **4.13**  you will need to implement a [child_record_callback](../callbacks#list-sorting-child-record) in **mode 4**, otherwise no records will be shown in backend.               |
```

Author: What has changed here?

Reviewer: Just what's bold ;). Typically only the version itself is bold, at least that's how I used it so far.

Author: Sorry, temporary blindness seems to have kicked in.

| [header_callback](../callbacks#list-sorting-header) | Callback function (`array`) | These functions will be called when the header fields (sorting mode 4 only) are created. Please specify as `['Class', 'Method']` or use an anonymous function.
| child_record_class    | CSS class (`string`)             | Allows you to add a CSS class to the parent view elements.                                                                                                                                                                                                                                                                                                                                             |

{{% notice tip %}}

> Since Contao **4.13** there are class constants available for the various sorting modes and flags. You find the constants in `Contao\DataContainer`, e.g.

Reviewer: ```suggestion
Since Contao **4.13** the `Contao\DataContainer` class provides constants for the various sorting modes and flags, e.g.
```
